src/EditorFeatures/Core/ExtractInterface/AbstractExtractInterfaceCommandHandler.cs

```diff
@@ -78,6 +78,7 @@ public bool ExecuteCommand(ExtractInterfaceCommandArgs args, CommandExecutionCon
                         return;
                     }
 
+                    await _threadingContext.JoinableTaskFactory.SwitchToMainThreadAsync();
```

Comment:
Should we just make that ConfigureAwait(true) above so that way we're not taking an extra jump to the (potentially full) thread pool before coming back?

Reply:
Sure. I wasn't sure pros/cons, but I'm happy to do that.